### ISSUE TITLE
Adjusting "No new activity" highlight colour based on last seen

### ISSUE THREAD

**Issue by ninja6o4** [HandyGuySoftware/dupReport#85]
Would be great if DR had some logic built in to highlight "No new activity" in different colours based on priority. Currently, if I run the report twice in a row, every single entry in the report highlights red, even though most are seen 0 days ago.

As an example, Crashplan sort of uses 0-2 days as no highlight, 3-6 days yellow highlight, 7+ days red highlight.

**Comment by HandyGuySoftware** [comment]
Sorry for the delay, been on the road this week. I'll take a look and see if it's something simple. If not, I'll work on it as soon as I get back on Tuesday.

**Comment by HandyGuySoftware** [comment]
@ninja6o4 , I pushed the Issue_85 branch to try to address this. What you are asking for is already managed in the stand-alone "last-seen" report option. I simply modified the code to use the same options for the standard report's last seen display. See the description (in the readme.md file for this branch) for the [report] lastseen* set of options for how it works. try it out and let me know if that's what you had in mind.
